lisa/sut_orchestrator/azure/common.py

```diff
@@ -192,3 +194,14 @@ def get_environment_context(environment: Environment) -> EnvironmentContext:
 def wait_operation(operation: Any) -> Any:
     # to support timeout in future
     return operation.wait()
+
+
+def create_storage_account_container_ifnotexists(
```

Comment:
How about create_storage_blob_container()? We can document it's a noop if it exists. Finally, how about introduce the patch when the usage is also ready? That way we can review the whole concept/idea better :)

Reply:
@glima `create_storage_blob_container ` sounds better. I will make this update. However, I didn't get what you mean by "introduce the patch when the usage is also ready?" Can you please explain?

Reply:
There's no caller site for the new function. What's the intended use?

Reply:
If it's something internal you could show me that off-band :)

Reply:
@glima What is the best way to document this function? I don't see any docstrings for the functions in `common.py`. Are these documented someplace else?

Reply:
LISA is pretty much undocumented, as we speak (at least internally). But please put in docstrings, the infra consuming them will come as soon as we have some time :)